packages/bundle/src/adaptiveCards/hooks/internal/useParseAdaptiveCardJSON.js

```diff
@@ -1,9 +1,9 @@
 import { hooks } from 'botframework-webchat-component';
-import { useCallback } from 'react';
+import { useCallback, useMemo } from 'react';
 
 import useAdaptiveCardsPackage from '../useAdaptiveCardsPackage';
 
-const { useDirection } = hooks;
+const { useDirection, useStyleOptions } = hooks;
```

Comment:
Does this need a `defaultStyleOptions.js` change?

Reply:
Discussed offline. We will do a change when we work on type definition file.